ebuild-writing/common-mistakes/text.xml

```diff
@@ -428,9 +428,14 @@ SLOT="0"
 
 <p>
 Make sure the <c>DESCRIPTION</c> is not overly long. Good descriptions will
-describe the main function of the package in a sentence.
+describe the main function of the package in a sentence. Don't repeat package
+name within <c>DESCRIPTION</c> pointlessly, for example avoid:
```

Comment:
I'm not a native speaker, but I think the sentence would be clearer with a definite article:
```suggestion
describe the main function of the package in a sentence. Don't repeat the
package name within <c>DESCRIPTION</c> pointlessly, for example avoid:
```